spec/acceptance/grafana_folder_spec.rb

```diff
@@ -139,7 +139,7 @@ class { 'grafana':
       end
 
       it 'folder contains dashboard' do
-        shell('curl --user admin:admin http://localhost:3000/api/dashboards/uid/example-dashboard') do |f|
+        shell('curl --user admin:admin http://localhost:3000/api/search?query=example-dashboard') do |f|
```

Comment:
Why the search query instead of uid since the uid is now predictable?

Reply:
The uid is only predictable when creating new dashboards. Even though I changed the behaviour for new and the old test works for now, I still consider it a more stable solution to not rely on uid=slug. The search works with the name, no matter what is in the uid field.